pkg/ingester/user_state.go

```diff
@@ -183,6 +200,8 @@ func (u *userState) getSeries(metric model.Metric, cfg *UserStatesConfig) (model
 	series = newMemorySeries(metric)
 	u.fpToSeries.put(fp, series)
 	u.index.add(metric, fp)
+	memSeriesCreatedTotal.WithLabelValues(u.userID).Inc()
```

Comment:
Mentioned in Slack already: I would put this next to the `Event().Log()` line that serves a similar purpose.